angular/core/config/material.coffee

```diff
@@ -1,6 +1,16 @@
 angular.module('loomioApp').config ($mdThemingProvider, $mdAriaProvider) ->
   theme = window.Loomio.theme
-  $mdThemingProvider.theme('default')
-    .primaryPalette(theme.primary_palette, theme.primary_palette_config)
-    .accentPalette(theme.accent_palette, theme.accent_palette_config);
+
+  if theme.custom_primary_palette
```

Comment:
I'm pretty skeptical that we need this much logic here. Can we get away with
```
$mdThemingProvider.theme('default')
  .primaryPalette(theme.primary_palette, theme.primary_palette_config)
  .accentPalette(them.accent_palette, theme.accent_palette_config)
```

and ensuring that those theme values are correct based on ENV on the ruby side?

Reply:
yea, we can do that.